spec/lib/coach/request_serializer_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'spec_helper'
+require 'active_support/core_ext/object/try'
 require 'coach/request_serializer'
 
 describe Coach::RequestSerializer do
-  describe '.apply_header_rule' do
+  describe "#apply_header_rule" do
```

Comment:
Hey, this should remain `.apply_header_rule` as it's a class method of `RequestSerializer`, in comparison to the below `serialize` which is an instance method.